src/metabase/driver.clj

```diff
@@ -963,6 +963,16 @@
   dispatch-on-initialized-driver
   :hierarchy #'hierarchy)
 
+(defmulti column-name-length-limit
+  "Return the maximum number of characters allowed in a column name, or `nil` if there is no limit."
```

Comment:
Perhaps this is a bit pedantic, but we could mention in the docstring for `column-name-length-limit` and `table-name-length-limit` that this is the maximum number of _single-byte_ characters, not characters in general. We currently use `u/slufigy` to convert non-ASCII characters to ASCII, but there's a future where we forget to do this.

Reply:
That's a good call out, this detail bothered me as well.